config/jobs/kubernetes/kops/kops-periodics-grid.yaml

```diff
@@ -1682,30 +1830,7282 @@ periodics:
+    - command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --cluster=e2e-kops-cilium-flatcar-k18.test-cncf-aws.k8s.io
+      - --deployment=kops
+      - --kops-ssh-user=core
+      - --env=KUBE_SSH_USER=core
+      - --env=KOPS_DEPLOY_LATEST_URL=https://storage.googleapis.com/kubernetes-release/release/stable-1.18.txt
+      - --env=KOPS_KUBE_RELEASE_URL=https://storage.googleapis.com/kubernetes-release/release
+      - --env=KOPS_RUN_TOO_NEW_VERSION=1
+      - --extract=release/stable-1.18
+      - --ginkgo-parallel
+      - --kops-args=--networking=cilium
+      - --kops-image=075585003325/Flatcar-stable-2512.2.0-hvm
+      - --kops-priority-path=/workspace/kubernetes/platforms/linux/amd64
+      - --kops-version=https://storage.googleapis.com/kops-ci/bin/latest-ci-updown-green.txt
+      - --provider=aws
+      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort|Services.*rejected.*endpoints|Services.*affinity
+      - --timeout=60m
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20200730-e636ee2-1.18
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-grid, kops-distro-flatcar, kops-k8s-1.18
+    testgrid-tab-name: kops-grid-cilium-flatcar-k18
+
+# {"cloud": "aws", "distro": "flatcar", "k8s_version": "1.18", "kops_version": "1.18", "networking": "cilium"}
+- name: e2e-kops-grid-cilium-flatcar-k18-ko18
+  cron: '40 12 * * 2'
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  decorate: true
+  decoration_config:
+    timeout: 90m
+  spec:
+    containers:
+    - command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --cluster=e2e-kops-cilium-flatcar-k18-ko18.test-cncf-aws.k8s.io
+      - --deployment=kops
+      - --kops-ssh-user=core
+      - --env=KUBE_SSH_USER=core
+      - --env=KOPS_DEPLOY_LATEST_URL=https://storage.googleapis.com/kubernetes-release/release/stable-1.18.txt
+      - --env=KOPS_KUBE_RELEASE_URL=https://storage.googleapis.com/kubernetes-release/release
+      - --env=KOPS_RUN_TOO_NEW_VERSION=1
+      - --extract=release/stable-1.18
+      - --ginkgo-parallel
+      - --kops-args=--networking=cilium
+      - --kops-image=075585003325/Flatcar-stable-2512.2.0-hvm
+      - --kops-priority-path=/workspace/kubernetes/platforms/linux/amd64
+      - --kops-version=https://storage.googleapis.com/kops-ci/markers/release-{kops_version}/latest-ci-updown-green.txt
+      - --provider=aws
+      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort|Services.*rejected.*endpoints|Services.*affinity
+      - --timeout=60m
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20200730-e636ee2-1.18
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-grid, ko
```

Comment:
Is this flag a valid kops flag?

Reply:
Doh!  Great point!

We need to keep the cluster name short; I totally forgot this was passed verbatim to kops - will fix!

Nice catch :-)